packages/kobber-components/src/wiki-list/List.ts

```diff
@@ -0,0 +1,55 @@
+import { LitElement, css, html, unsafeCSS } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import "./ListItem";
+import { consume } from "@lit/context";
+import { Theme } from "../utils/theme-context.types";
+import { themeContext } from "../utils/theme-context";
+
+export const customElementName = "kobber-wiki-list";
+
+type ListDirection = "vertical" | "horizontal";
+
+/**
+ * Vertical or horizontal display of elements.
+ */
+@customElement(customElementName)
+export class List extends LitElement {
+  @consume({ context: themeContext, subscribe: true })
+  theme?: Theme;
+
+  @property()
+  direction: ListDirection = "vertical";
+
+  public override connectedCallback(): void {
+    super.connectedCallback();
+    this.setAttribute("role", "menu");
```

Comment:
Er `menu` rett role? I følge [ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/) er "A menu is usually opened, or made visible, by activating a menu button", mens "A menu that is visually persistent is a menubar." (Hvis du endrer til menubar, så husk at "If a menubar is vertically oriented, it has aria-orientation set to vertical. The default value of aria-orientation for a menubar is horizontal.").

Uansett (fra [ARIA Authoring Practices Guide](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/#wai-ariaroles,states,andproperties)):
- An element with role menu either has: 
  - aria-labelledby set to a value that refers to the menuitem or button that controls its display.
  - A label provided by aria-label.
- If a menubar has a visible label, the element with role menubar has aria-labelledby set to a value that refers to the labelling element. Otherwise, the menubar element has a label provided by aria-label.

Reply:
Men at både menu og menubar må ha en beskrivende tekst er litt slitsomt